lambda/cfnresponse.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright 2016 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.
+#  This file is licensed to you under the AWS Customer Agreement (the "License").
+#  You may not use this file except in compliance with the License.
+#  A copy of the License is located at http://aws.amazon.com/agreement/ .
+#  This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+#  See the License for the specific language governing permissions and limitations under the License.
+
+import requests
+import json
+
+SUCCESS = "SUCCESS"
+FAILED = "FAILED"
+
+
+def send(
+    event, context, responseStatus, responseData, physicalResourceId=None, noEcho=False
+):
+    responseUrl = event["ResponseURL"]
+
+    print(responseUrl)
+
+    responseBody = {}
+    responseBody["Status"] = responseStatus
+    responseBody["Reason"] = (
+        "See the details in CloudWatch Log Stream: " + context.log_stream_name
+    )
+    responseBody["PhysicalResourceId"] = physicalResourceId or context.log_stream_name
+    responseBody["StackId"] = event["StackId"]
+    responseBody["RequestId"] = event["RequestId"]
+    responseBody["LogicalResourceId"] = event["LogicalResourceId"]
+    responseBody["NoEcho"] = noEcho
+    responseBody["Data"] = responseData
+
+    json_responseBody = json.dumps(responseBody)
+
+    print("Response body:\n" + json_responseBody)
+
+    headers = {"content-type": "", "content-length": str(len(json_responseBody))}
+
+    try:
+        response = requests.put(responseUrl, data=json_responseBody, headers=headers)
+        print("Status code: " + response.reason)
+    except Exception as e:
```

Comment:
Have a similar situation going on here... Exceptions just get swallowed instead of captured in the failure metric. I realize this is Amazon's code, but it's still not a great practice...

Reply:
added a custom